src/components/DraggableItem.tsx

```diff
@@ -21,15 +22,17 @@ interface Props {
   index: number;
 }
 
-const StyledListElement = styled.li`
-  list-style: none;
-  padding: 0;
-  display: flex;
-  align-items: center;
-  &[data-has-handle="false"] {
-    cursor: grab;
-  }
-`;
+const StyledListElement = styled("li", {
+  base: {
+    listStyle: "none",
+    display: "flex",
+    alignItems: "center",
+    padding: "0",
+    "&[data-has-handle='false']": {
+      cursor: "grab",
+    },
```

Comment:
Gir det mer mening å skrive om dette til `&:has([data-drag-handle])`, og heller sette data-attributt på drag-handle?

Reply:
Er ikke poenget at hele elementet kan dras i når det mangler drag-handle? eller misforstår jeg?

Reply:
Jo, det stemmer! Da må du eventuelt endre til `&:not(:has(> [data-drag-handle])`, eller noe i den duren. Tenkte det kanskje ville vært ryddigere, men det er jo ikke nødvendigvis sant!

Reply:
Vet ikke om jeg syntes det blir noe ryddigere 😅